include/util/misc.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 CodiLime
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#pragma once
+
+namespace veles {
+namespace util {
+
+template<typename T>
+T bytesToIntLe(const uint8_t* bytes, size_t num_bytes) {
+  assert(num_bytes <= sizeof(T));
+  T res = T();
+  for (int i = 0; i < num_bytes; ++i) {
+    res |= static_cast<T>(bytes[i]) << (i * 8);
+  }
+  return res;
+}
+
+template<typename T>
+uint8_t* intToBytesLe(T val, size_t num_bytes) {
```

Comment:
Do we want this to alloc memory? IMO would be better to store bytes to caller-allocated storage.